src/Azure/Akka.Streams.Azure.ServiceBus/BusClientWrapper.cs

```diff
@@ -1,64 +1,43 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
-using Microsoft.ServiceBus.Messaging;
+using Azure.Messaging.ServiceBus;
 
 namespace Akka.Streams.Azure.ServiceBus
 {
     internal interface IBusClient
```

Comment:
This interface and their concrete implementations are ment to be a facade for the old ServiceBus clients, it is needed because the old Microsoft implementation requires us to use 3 different kind of clients and we wanted to unify them into a single facade.

Since Microsoft unified all 3 clients into a single sender or receiver, you should remove this interface and calls the sender/receiver client directly instead.